helm/prometheus-rules/templates/alerting-rules/vpa-too-many-restart.rules.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: grafana.all.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: vpa
+    rules:
+    - alert: VpaComponentTooManyRestarts
+      annotations:
+        description: This pages when one of the vpa's component has restarted too much over the last 10min.
+        opsrecipe: vpa-component-too-many-restarts/
+      expr: |
+        1 - sum(increase(kube_pod_container_status_restarts_total{container=~"recommender|updater|admission-controller"}[10m])) by (container, cluster_id, cluster_type, customer, installation, pipeline, provider, region)/100
+          or
+        1 - sum(increase(kube_pod_container_status_restarts_total{container="vertical-pod-autoscaler-app"}[10m])) by (container, cluster_id, cluster_type, customer, installation, pipeline, provider, region)/100
+          < 0.98
+      for: 10m
+      labels:
+        area: managedservices
+        cancel_if_apiserver_down: "true"
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_scrape_timeout: "true"
+        cancel_if_outside_working_hours: "false"
```

Comment:
Don't we decide to be alerted only during working hours for this one ?
The SLO one will wake up oncall night.